docs/reference/authentication.md

```diff
@@ -18,7 +18,7 @@ has_children: false
 ### User Authentication
 
 lakeFS authenticates users from a built-in authentication database, or,
-optionally, from a configured LDAP server.
+optionally.
```

Comment:
```suggestion
optionally, from an external authentication database.
```

In any case, we cannot say "lakeFS authenticates users from a built-in authentication database, or, optionally.".  Either drop everything and say "lakeFS authenticates users from a built-in authentication database" or mention an external authentication server somehow.

Reply:
Done Thanks!